src/features/offer/components/OfferBody/OfferBody.analytics.native.test.tsx

```diff
@@ -153,6 +212,32 @@ describe('<OfferBody /> - Analytics', () => {
       from: 'offer',
     })
   })
+
+  describe('When wipEnableMultivenueOffer feature flag activated', () => {
+    beforeEach(() => {
+      mockUseOffer.mockImplementation(() => ({
+        data: {
+          ...offerResponseSnap,
+          subcategoryId: SubcategoryIdEnum.LIVRE_PAPIER,
+          extraData: {
+            ean: '2765410054',
+          },
+        },
+      }))
```

Comment:
Idem que plus haut, je pense que tu peux faire plus simple
```suggestion
      mockUseOffer.mockReturnValue({
        data: {
          ...offerResponseSnap,
          subcategoryId: SubcategoryIdEnum.LIVRE_PAPIER,
          extraData: {
            ean: '2765410054',
          },
        },
      })
```